README.md

```diff
@@ -78,6 +78,11 @@ To reload the config
 ./build/src/swaync-client -R
 ```
 
+To reload css after changes
```

Comment:
Markdown requires a line between code blocks and text. Other than that it LGTM :)

Reply:
fixed with force push